benchmark/mnist/mnist.yaml

```diff
@@ -48,3 +48,40 @@ selection_strategy:
   storage_backend: "database"
   limit: -1
   tail_triggers: 0
+evaluation:
```

Comment:
there are two mnist.yaml files. One here and one in `modynclient` directory.

The one in `modynclient` is not used anywhere in code. So I removed that one but add the evaluation section to this one.